tests/examples/deploy.pp

```diff
@@ -1,7 +1,21 @@
-include jboss
+class { 'jboss':
+  product       => 'jboss-as',
+  version       => '7.1.1.Final',
+  download_url  => 'http://download.jboss.org/jbossas/7.1/jboss-as-7.1.1.Final/jboss-as-7.1.1.Final.zip',
```

Comment:
!fix This should be automatically resolved, remove this address from example file